cypress/page/menu-content.page.ts

```diff
@@ -3,7 +3,9 @@ class MenuContentPage {
   private menuContentPageURL: string
 
   constructor() {
-      this.tShirtMenu = "#block_top_menu > ul > li:nth-child(3) > a";
+      /* Now the selector searchs the last li in Ul and and a label
+        that is inside the li */
+      this.tShirtMenu = "#block_top_menu > ul > li:last-child a";
```

Comment:
according to what we talked in last class, this selector can use the title attribute of the `a` tag